### ISSUE TITLE
DICOM Microscopy extension using improper Accept Header

### ISSUE THREAD

**Issue by Malik-Adil** [OHIF/Viewers#624]
I have to Download the image form "https://wsi.orthanc-server.com/orthanc" and upload on my local server. While trying to Show in "DICOM Microscopy Viewer Example" it gives me Bad Request Error 

![Screenshot from 2019-07-03 10-12-55](https://user-images.githubusercontent.com/20414066/60565010-48d57e00-9d7b-11e9-87b1-c570bacec9da.png)

Request Send on Server in Network is

![Screenshot from 2019-07-03 10-13-25](https://user-images.githubusercontent.com/20414066/60565028-54c14000-9d7b-11e9-99eb-409a65da35a8.png)

and the response is
![Screenshot from 2019-07-03 10-14-59](https://user-images.githubusercontent.com/20414066/60565039-5f7bd500-9d7b-11e9-920c-04f345bec57c.png)

Kindly tell me where I do wrong 

Thanks

**Comment by swederik** [comment]
I think you aren't doing anything wrong. It looks like this is an issue with the accept header string. It looks like I broke it in this in this PR: https://github.com/dcmjs-org/dicom-microscopy-viewer/pull/29

I have filed an issue here (https://github.com/dcmjs-org/dicom-microscopy-viewer/issues/32) and made a PR to dicom-microscopy-viewer to address the issue (https://github.com/dcmjs-org/dicom-microscopy-viewer/pull/33).

Thanks for the report!
